guides/common/modules/con_api-cheat-sheet.adoc

```diff
@@ -2,35 +2,19 @@
 = API cheat sheet
 
 You can review the following examples of how to use the {ProjectName} API to perform various tasks.
-You can use the API on {ProjectServer} via HTTPS on port 443 or on {SmartProxyServer} via HTTPS on port 8443.
+You can use the API on {ProjectServer} via HTTPS on port 443.
 
-[IMPORTANT]
-====
-ifdef::foreman,katello[]
-Calling the API through {SmartProxyServer} has been deprecated since {Project} 3.12.
-endif::[]
-ifdef::satellite[]
-Calling the API through {SmartProxyServer} has been deprecated since {Project} 6.16.
-endif::[]
-ifdef::orcharhino[]
-Calling the API through {SmartProxyServer} has been deprecated since {Project} XXX.
-endif::[]
-Migrate your integrations to call the API through {ProjectServer}.
-====
-
-You can address these different port requirements within the script itself.
-For example, in Ruby, you can specify the {ProjectServer} and {SmartProxyServer} URLs as follows:
+For example, in Ruby, you can specify the {ProjectServer} URL as follows:
 
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 url = 'https://_{foreman-example-com}_/api/v2/'
-{smart-proxy-context}_url = 'https://_{smartproxy-example-com}_:8443/api/v2/'
 ifdef::katello,orcharhino,satellite[]
 katello_url = 'https://_{foreman-example-com}_/katello/api/v2/'
 endif::[]
 ----
 
-For the host that is registered to {ProjectServer} or {SmartProxyServer}, you can determine the correct port required to access the API from the `/etc/rhsm/rhsm.conf` file, in the port entry of the `[server]` section.
+For the host that is registered to {ProjectServer}, you can determine the correct port required to access the API from the `/etc/rhsm/rhsm.conf` file, in the port entry of the `[server]` section.
```

Comment:
I am wondering if this statement is needed any longer as port 443 would be the one to be used by default.

Reply:
Thanks for the hint, let's try dropping it too.